js/Events/logInEvents.js

```diff
@@ -1,44 +1,47 @@
 const addEventsLogIn = function (){
-
     const submitFormEvent = function (event) {
     
-    event.stopPropagation();
-   event.preventDefault();
+        event.stopPropagation();
+        event.preventDefault();
 
-    const userNameInput = document.querySelector('#userNameInput');
-    const passwordInput = document.querySelector('#passwordInput');
+        const userNameInput = document.getElementById('userNameInput');
+        const passwordInput = document.getElementById('passwordInput');
+        const userNameValue = userNameInput.value;
+        const passwordValue = passwordInput.value;
```

Comment:
does it bring any value to store both the inputs and the input values in constants?

Reply:
Not really, I was thinking that maybe later I will use again the inputs.